build/build-bundle.js

```diff
@@ -152,6 +151,12 @@ async function build(entryPath, distPath, opts = {minify: true}) {
           import Audit from '${require.resolve('../lighthouse-core/audits/audit.js')}';
           export {Audit};
         `,
+        // Most node 'url' polyfills don't include the WHATWG `URL` property, but
+        // that's all that's needed, so make a mini-polyfill.
+        // @see https://github.com/GoogleChrome/lighthouse/issues/5273
+        // TODO: remove when not needed for pubads (https://github.com/googleads/publisher-ads-lighthouse-plugin/pull/325)
+        // and robots-parser (https://github.com/samclarke/robots-parser/pull/23)
+        'url': 'export const URL = globalThis.URL;',
```

Comment:
exporting globals is annoying. Open to nicer ways of doing this

Reply:
Hmm I was expecting this to cause issues in CI because we use `fileURLToPath` in `root.js`:
https://github.com/GoogleChrome/lighthouse/blob/da73d4ae0455576a196b6bba776820f29fc2c4f1/root.js#L24

Reply:
That's what I was referring to in https://github.com/GoogleChrome/lighthouse/pull/13519#issuecomment-1006965651, it doesn't work in the current version either. Either nothing is calling it or at least not calling it with an `importMeta`. If/when we do need that (if we don't transform it out of the bundle in a different way) we'll need to find a different polyfill because rollup-plugin-polyfill-node doesn't have `fileURLToPath` either :/

Reply:
robots parser already shipped a new version (3.0.0) with that PR.   pubads didnt yet